handlers/main.yml

```diff
@@ -24,3 +24,12 @@
     become: True
     command: systemctl daemon-reload
     notify: restart ldap-groups
+
+  - name: reload systemd-postfix
+    become: True
+    command: systemctl daemon-reload
```

Comment:
systemd module has a daemon_reload option

Reply:
Right. Fixed in f9ebbd4.